packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -373,13 +373,27 @@ export default class RunPanelStore {
 
     @action.bound
     showErrorMessage(data) {
-        const { journal } = this.root_store;
+        const { journal, ui } = this.root_store;
+
         journal.onError(data);
         if (journal.journal_filters.some(filter => filter === message_types.ERROR)) {
             this.setActiveTabIndex(2);
+        } else {
+            ui.addNotificationMessage(journalError(this.switchToJournal));
+            ui.removeNotificationMessage({ key: 'bot_error' });
         }
     }
 
+    @action.bound
+    switchToJournal() {
+        const { journal, ui } = this.root_store;
+        journal.journal_filters.push(message_types.ERROR);
+        this.setActiveTabIndex(2);
```

Comment:
Could we create constants for the indexes so it'll be easier for other developers to understand which tab is e.g. `2`?